core/src/blind_rotation/key.rs

```diff
@@ -0,0 +1,105 @@
+use backend::{Backend, FFT64, Module, ScalarZnx, ScalarZnxAlloc, ScalarZnxToRef, Scratch, ZnxView, ZnxViewMut};
+use sampling::source::Source;
+
+use crate::{Distribution, FourierGLWESecret, GGSWCiphertext, Infos, LWESecret};
+
+pub struct BlindRotationKeyCGGI<B: Backend> {
+    pub(crate) data: Vec<GGSWCiphertext<Vec<u8>, B>>,
```

Comment:
Why is `Vec<GGSWCiphertext<Vec<u8>, B>>` not abstract over Data store?

Reply:
Will change it